src/mono/netcore/sample/wasm/WasmSample.csproj

```diff
@@ -25,10 +25,14 @@
              Targets="Build;Publish"/>
   </Target>
 
-  <UsingTask TaskName="WasmAppBuilder" 
+  <UsingTask TaskName="WasmAppBuilder"
              AssemblyFile="$(ArtifactsBinDir)WasmAppBuilder\$(Configuration)\$(NetCoreAppCurrent)\publish\WasmAppBuilder.dll"/>
 
   <Target Name="BuildApp" DependsOnTargets="RebuildWasmAppBuilder;Build">
+  <ItemGroup>
+      <WasmFilesToIncludeInFileSystem Include="@(ContentWithTargetPath)" />
+      <WasmFilesToIncludeInFileSystem Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.BuildReference)' == 'true'" />
```

Comment:
This was needed in the mobile targets just for ProjectReferences, do we have any in the `WasmSample`?

Reply:
I don't think so. Let me take that out.